contingency/contingency-api/src/main/java/com/powsybl/contingency/BusContingency.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
I think you forgot SPDX-License-Identifier: MPL-2.0

Reply:
Yes, done.